docs/Makefile

```diff
@@ -5,8 +5,8 @@
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = .
-BUILDDIR      = build
-PYTHON3       = python3
+BUILDDIR      = _build
```

Comment:
What is the reason for prefixing the underscore to this directory?

Reply:
Few reasons:
1. Build directory is configured in docs/make.bat as _build (inconsistent)
2. _build is used in other projects
3. _build will appear on the top of the directory list in IDE

I agree the .gitignore should be changed for this. Which I will do.